app/jobs/send_school_consent_notification_job.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class SendSchoolConsentNotificationJob < ApplicationJob
+  queue_as :notifications
+
+  def perform(session)
```

Comment:
It feels like there's quite a lot of logic happening here, it'd be good to extract it into a separate class and have a very thin job.